docs/Guide-technique-services-sociétaux.md

```diff
@@ -279,16 +348,16 @@ Un autre fichier a été utilisé, nommé `titres_raccourcis.yaml` qui permet de
 
 ## Limites du calcul
 
 - Il est possible que le fichier dont nous disposons pour décomposer les intensités carbone des différentes branches ne soit pas adapté au fichier du SDES. (**nomenclature NAF vs CPA**) : est ce que le chiffre d'affaire couvre les même périmètres que pour les données du SDES (Territoire française, nomenclature ?). Par ailleurs, ces chiffres d'affaire inclut également ce qui est vendu et **exporté**, on fait donc implicitement l'hypothèse que les chiffre d'affaire correspond à 50% de ventes sur le territoire français et 50% à l'étranger.
-- **Certaines valeurs de ce fichier sont "secrètes"** : les budgets associés à la défense ne sont volontairement pas public. Dans une première approche nous avons omis ces postes pour la décomposition des postes d'émission en supposant qu'elles étaient également secrètes pour le SDES. Finalement il semble qu'elles sont connues a des fins statistiques et l'agrégation proposée ne permet pas de remonter jusqu'à ces chiffres. Nous n'avons pour le moment pas d'autres pistes pour résoudre les problèmes liés à cette approche par chiffre d'affaire. Il faudrait explorer d'autres méthodes afin de désagréger ces branches.
-- Une autre limite importante à cette méthode de calcul, qui concerne plus généralement l'approche macro-économique, est le **caractère homogène ou non du contenu de chaque branche**. Par exemple, dans le secteur de la construction, les émissions carbone associées à la construction d'un local en brique, d'une tiny house ou de prestations de finitions (peinture) pour un même investissement seront très hétérogènes. Il s'agira peut-être de creuser certains postes importants des services sociétaux afin de limiter les incertitudes (changements d'approche ?).
+- **Certaines valeurs de ce fichier sont "secrètes"** : les budgets associés à la défense ne sont volontairement pas rendus publics. Dans une première approche nous avons omis ces postes pour la décomposition des postes d'émission en supposant qu'elles étaient également secrètes pour le SDES. Finalement il semble qu'elles sont connues à des fins statistiques et l'agrégation proposée ne permet pas de remonter jusqu'à ces chiffres. Nous n'avons pour le moment pas d'autres pistes pour résoudre les problèmes liés à cette approche par chiffre d'affaire. Il faudrait explorer d'autres méthodes afin de désagréger ces branches.
+- Une autre limite importante à cette méthode de calcul, qui concerne plus généralement l'approche macro-économique, est le **caractère homogène ou non des produits de chaque branche**. Par exemple, dans le secteur de la construction, les émissions carbone associées à la construction d'un local en brique, d'une tiny house ou de prestations de finitions (peinture) pour un même investissement seront très hétérogènes. Il s'agira peut-être de creuser certains postes importants des services sociétaux afin de limiter les incertitudes (changements d'approche ?).
```

Comment:
Je comprends bien l'hétérogénéité des produits mais plus que cette hétérogénéité à creuser, c'est pour moi l'aspect "sociétaux" des produits qu'il faut vérifier car un code CPA peut à la fois regrouper des produits rentrant dans la classification "services sociétaux" et d'autres n'y rentrant pas

Reply:
Yes, c'est une limite à ajouter dans ce cas :)